drivers/clock_control/Kconfig.stm32l4_l5_wb

```
@@ -9,43 +9,59 @@ config CLOCK_STM32_PLL_M_DIVISOR
 	int "PLL divisor"
 	depends on CLOCK_STM32_SYSCLK_SRC_PLL
 	default 1
-	range 1 8
+	range 1 8 if SOC_SERIES_STM32L4X || SOC_SERIES_STM32WBX
+	range 1 16 if SOC_SERIES_STM32L5X
 	help
-	  PLL divisor, allowed values: 1-8. With this ensure that the PLL
-	  VCO input frequency ranges from 4 to 16MHz.
+	  PLL divisor,
+	  L4: allowed values: 1-8. PLL VCO input ranges from 4 to 16MHz
```

Minor fix (apply everywhere)
```suggestion
	  L4: allowed values: 1-8. PLL VCO input range from 4 to 16MHz
```

---

This is fine if reading "range" as a verb.